app/src/main/java/org/oppia/android/app/devoptions/ForceCrashListener.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.android.app.devoptions
+
+/** Listener for force crashing the app. */
+interface ForceCrashListener {
```

Comment:
Perhaps: ``ForceCrashButtonClickListener`` might be a bit clearer as to what's being listened for. At the moment, this name could also be construed as listening for when a force crash happens, not when we want it to happen.

Reply:
Refactored.